### ISSUE TITLE
Process conftool output

### ISSUE THREAD

**Issue by waynegraham** [waynegraham/sched-api#1]
The output from conftool needs massaging to use the Excel import for Sched. 

**Presentation 1:**
[p1 title]
[p1 abstract]
*Presenter(s):* [p1 authors]

**Presentation 2:**
[p2 title]
[p2 abstract]
**Presenter(s):** [p2 authors]

...

**Comment by waynegraham** [comment]
The format should be:

```
<p>
Presentation 1:<br /><strong>[p1 title]</strong><br />[p1 abstract]<br/>Presenter(s): [p1 authors]</p>
```

**Comment by waynegraham** [comment]
closed in 704030f51feaa4743ca26a6a8bd87bebaeb302e1
